aws/table_aws_s3_bucket.go

```diff
@@ -26,77 +28,77 @@ func tableAwsS3Bucket(_ context.Context) *plugin.Table {
 		// list call.
 		HydrateConfig: []plugin.HydrateConfig{
 			{
-				Func: getBucketLocation,
-				Tags: map[string]string{"service": "s3", "action": "GetBucketLocation"},
+				Func: getBucketRegion,
+				Tags: map[string]string{"service": "s3", "action": "HTTPHeadBucket"},
```

Comment:
```suggestion
				Tags: map[string]string{"service": "s3", "action": "HeadBucket"},
```

Our `action` tags are usually based on the AWS API name, so I think better to match what the API uses (S3 API names are a bit weird sometimes though, so not sure if `HTTPHeadBucket` is actually the API name?)

Reply:
That's a fair point, maybe `HeadBucket` would be a good name to match the S3 API name, https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadBucket.html?